### PR TITLE
Update CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by wesleywesley**
Now that microsemi/switchtec-user repository equipped with travis-ci,
we can ask contributor create pull request against this repo.

Update the "pull request" section, to ask contributors create PR
based on the devel branch.

Signed-off-by: Wesley Sheng <wesley.sheng@microchip.com>

### CODE REVIEW

**Comment by lsgunth** [comment]
Probably worth also adding text to ask contributors to follow kernel style and maybe run checkpath.

**Comment by wesleywesley** [comment]
@lsgunth 
Thx for review.
Updated the content to include run checkpatch.pl script.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ document and take heed of its wisdom.
...
 
-This project uses Travis to check that it always builds on all the supported
-platforms. However, Microsemi's IT department stubbornly refuses (for silly
-reasons) to turn it on for the main repository. Therefore, in order to ensure
-your contributions get tested, please submit all pull requests to
-[lsgunth/switchtec-user][2]. From there we will ensure your contribution
-gets merged into the main repository.
+Please set the base branch to microsemi/devel when create the pull request.
+Make sure the comparing changes inclue your bug fixes or new features, the
+patches has no obvious style problems and is ready for submission when
```
Comment (lsgunth) [inline CONTRIBUTING.md]: Should be "patches have no obvious..."

Also, I think you should state the actual style requirements here: we follow kernel style as documented [here](https://www.kernel.org/doc/html/latest/process/coding-style.html).

Running the checkpatch script should be a recommendation to ensure this. Especially seeing it's not included in this tree.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,12 +12,12 @@ document and take heed of its wisdom.
...
+check by checkpatch.pl script of the kernel tree.
```
Comment (lsgunth) [inline CONTRIBUTING.md]: You should say script of the "Linux Kernel tree".

Just "kernel" is ambiguous.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -12,12 +12,12 @@ document and take heed of its wisdom.
...
-reasons) to turn it on for the main repository. Therefore, in order to ensure
-your contributions get tested, please submit all pull requests to
-[lsgunth/switchtec-user][2]. From there we will ensure your contribution
-gets merged into the main repository.
+Please set the base branch to microsemi/devel when create the pull request.
+Make sure the comparing changes inclue your bug fixes or new features.
+It is highly recommended that check the patch by checkpatch.pl script of
+Linux Kernel tree, confirm no obvious style problems and is ready for
+submission. Once the patch go through our review and verification stages,
+it will be merged into microsemi/devel branch.
```
Comment (lsgunth) [inline CONTRIBUTING.md]: The english in every one of these sentences has gotten pretty broken. I suggest you get someone to edit it for you.

Also, you didn't add the description of the style we follow as I suggested in my previous review.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -12,12 +12,12 @@ document and take heed of its wisdom.
...
 
-This project uses Travis to check that it always builds on all the supported
-platforms. However, Microsemi's IT department stubbornly refuses (for silly
-reasons) to turn it on for the main repository. Therefore, in order to ensure
-your contributions get tested, please submit all pull requests to
-[lsgunth/switchtec-user][2]. From there we will ensure your contribution
-gets merged into the main repository.
+Please create the pull request basing the branch devel on this repo.
+Make sure the comparing changes include your bug fixes or new features.
+We follow kernel style as documented[here][1]. It is highly recommended
```
Comment (lsgunth) [inline CONTRIBUTING.md]: I think there should be  space between "documented" and "here".

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -12,12 +12,12 @@ document and take heed of its wisdom.
...
+that the patches run through the check of checkpatch.pl in the Linux
+Kernel tree with no obivous style problems. Once the patches go through
```
Comment (lsgunth) [inline CONTRIBUTING.md]: "It is highly recommended that the patches be checked with the checkpatch.pl script in the Linux Kernel tree."

**Comment by wesleywesley** [comment]
@Isgunth

Thx for review. 
Updated.

Regard,
Wesley

**Pull request merged**